File: 3279-alice-and-bob-playing-flower-game/alice-and-bob-playing-flower-game.cpp
class Solution {
public:
    long long flowerGame(int n, int m) {
        int e_n = 0;
        int e_m = 0;



        for(int i=1;i<=n;i++){
            if(i % 2 == 0) e_n++;
        }

        for(int  i=1;i<=m;i++){
            if(i % 2 == 0) e_m++;
        }

        return (1LL*e_n * (m-e_m)) + (1LL*e_m * (n-e_n));
    }
};

File: 3279-alice-and-bob-playing-flower-game/alice-and-bob-playing-flower-game.py
class Solution:
    def flowerGame(self, n: int, m: int) -> int:
        e_n = sum(1 for i in range(1,n+1) if i%2 == 0)
        o_n = n - e_n
        e_m = sum(1 for i in range(1,m+1) if i%2 ==0)
        o_m = m - e_m

        ans = (e_n * o_m) + (o_n * e_m)
        return ans